app/components/common/search-selector/index.js

```diff
@@ -17,8 +18,8 @@ const closeImage = require('assets/close.png');
 
 function getFilteredData(data, filter) {
   if (!filter) return data;
-  const filterUpper = filter.toUpperCase();
-  return data.filter((item) => item.name.toUpperCase().indexOf(filterUpper) > -1);
+  const filterUpper = deburr(filter.toUpperCase());
```

Comment:
I didn't know that [feature](https://lodash.com/docs/4.17.4#deburr) of lodash, really useful thanks!